// import { CustomError } from "../../utils/customError";
// import { loginRepository } from "@/src/repository/user/loginRepository";

export async function login (email: string, password: string ) {
    // try {
    return {id: "1", name: "edu"};
    // const user = await loginRepository(email);
    // if (user) {
    //     if (user.password === password) {
    //         return {id: user._id, name: user.name};
    //     }
    //     throw new CustomError("Senha incorreta!", 401);
    // }
    // throw new CustomError("O usuário não existe!", 404);
    // } catch (e: any) {
    //     throw e;
    // }
}
